fastcrypto-zkp/src/bn254/zk_login.rs

```diff
@@ -420,9 +423,9 @@ impl ZkLoginProof {
     /// Convert the Circom G1/G2/GT to arkworks G1/G2/GT
     pub fn as_arkworks(&self) -> Result<Proof<Bn254>, FastCryptoError> {
         Ok(Proof {
-            a: g1_affine_from_str_projective(self.a.clone())?,
-            b: g2_affine_from_str_projective(self.b.clone())?,
-            c: g1_affine_from_str_projective(self.c.clone())?,
```

Comment:
possibly why this takes too much time in flamegraph 🤔

Reply:
You mean the clone? The flamegraph shows that it's the subgroup check called by the `g*_from_str_projective` which takes almost all the time.